test/conformance/kuttl/policy-validation/cluster-policy/success/policy-1.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: vault-init-injector
+  annotations:
+    policies.kyverno.io/title: Inject vault init Container
+spec:
+  background: false
+  validationFailureAction: Audit
+  rules:
+  - name: inject-vault-sidecar
+    match:
+      resources:
+        kinds:
+        - Deployment
+    preconditions:
+      - key: "{{ request.object.spec.template.metadata.annotations.\"vault.k8s.corp.com/inject\"  || ''}}"
+        operator: Equals
+        value: "true"
+      - key: vault-init
+        operator: AnyNotIn
+        value: "{{ request.object.spec.template.spec.initContainers[].name || ['']}}"
```

Comment:
Needs to be updated with `any` for `match` and `all` for `preconditions[]`.

Reply:
Arf, this policy was copy pasted directly from the issue.

Reply:
Fixed